tests/utils/fakeredis_ext.py

```diff
@@ -109,6 +114,7 @@ def __init__(self, client):
             self.client = client
             self._sock = self._socket()
             self.pid = 1234
+            self.retry = Retry(NoBackoff(), 0)
```

Comment:
This change is the result of this change in the redis library: https://github.com/redis/redis-py/pull/1494/files#diff-3161b005adfa1f5266a9736bffb5ede9d14e2ab40937cb241bc16ca5647df79dR974. It now expects a retry attribute on all connections.